client/src/components/Reviews/components/Review.jsx

```diff
@@ -1,14 +1,65 @@
 // may add a diffrent file for helfult/report buttons
 import React from 'react';
 import styled from 'styled-components';
+import { formatDistanceToNow, parseISO } from 'date-fns';
 
 const StyledReview = styled.div`
   border: 2px solid black;
   margin: 10px;
 `;
+const Stars = styled.p`
+  padding-top: 10px;
+  float: left;
+  padding-left: 10px;
+`;
+const User = styled.p`
+  padding-top: 10px;
+  float: right;
+  padding-right: 10px;
+`;
+const Title = styled.h2`
+  padding-top: 10px;
+  font-size: 150%;
+`;
+const Buttons = styled.button`
+  background: none!important;
+  border: none;
+  cursor: pointer;
+`;
 
-function Review() {
-  return <StyledReview>review</StyledReview>;
+function Review({ review }) {
+  let date;
+  let rec = '';
+  function HandleReviewClick(e, type) {
+    e.preventDefault();
+    // send request to review.review_id, type (helpful or report)
+  }
+  function ParseData() {
+    date = formatDistanceToNow(parseISO(review.date));
+    if (review.recommend) {
+      rec = '✅';
+    }
+  }
+  ParseData();
+  return (
+    <StyledReview>
+      <Stars>{review.rating}</Stars>
+      {/* need to add fn to display stars */}
+      <User>
+        {`${rec} `}
```

Comment:
Don't need to use template literal if you're not doing any interpolation. This could just be `{rec}`.

Reply:
yeah, It's there to add in space after it since it wants each {} to be on its own line if I want to add a space I have to add it as a string, or at least that's the only way I found. though looking back I could do them all in one line if I'm doing interpolation.

Reply:
you know, I totally missed the space there. If it's all meant to be on one line visually anyways the easiest way might be to join them in one expression like you said. Makes it easier to read imo.